### PR TITLE
[INV-99]-Get-reviews-of-accommodation

### PR DESCRIPTION
**Pull request by blackseriesC63**
Screenshot 2024-11-27 at 11 09 58

https://github.com/user-attachments/assets/3a07254e-1f0f-42c1-84e5-db8840a7bb15

### CODE REVIEW

File: src/modules/accommodations/controllers/accommodations.controller.ts [hunk 1]
```diff
@@ -78,17 +78,20 @@ export class AccommodationController {
   }
 
   @Get(':id')
-  @ApiOperation({ summary: 'Get accommodation by ID' })
+  @ApiOperation({ summary: 'Get accommodation by ID with reviews' })
   @ApiResponse({
     status: 200,
-    description: 'Accommodation fetched successfully',
-    type: Accommodation,
```
Comment (NegatiV-64) [inline src/modules/accommodations/controllers/accommodations.controller.ts:85]: I think that type is missing here

File: src/modules/accommodations/services/accommodations.service.ts [hunk 2]
```diff
@@ -103,22 +103,45 @@ export class AccommodationService {
...
+      .where('accommodation.id = :id', { id })
+      .andWhere('accommodation.deletedAt IS NULL')
+      .getOne();
+
     if (!accommodation) {
       throw new NotFoundException(`Accommodation with ID ${id} not found`);
     }
-    return accommodation;
+
+    const transformed = {
```
Comment (NegatiV-64) [inline src/modules/accommodations/services/accommodations.service.ts:123]: I don't think that it's the best naming for this

**Review by NegatiV-64 — CHANGES_REQUESTED** [review]
Please add tests

**Comment by blackseriesC63** [comment]
Screenshot 2024-11-27 at 11 30 56
Tests successfully passed

**Review by NegatiV-64 — APPROVED** [review]

**Pull request merged**